docs/CONTRIBUTING.md

```diff
@@ -1,46 +1,61 @@
 # General Things:
-Use python syntax/features. If sage features are necessary, import them via import statements
+
+Use python syntax/features. If sage features are necessary, import them via
+import statements
 
 ## GIT Conventions
+
 ### Commits
-To contribute to this project please follow this subset of [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). These are some examples
-Type
-Must be one of the following:
+
+To contribute to this project please follow this subset of
+[conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). These
+are some examples Type Must be one of the following:
+
 - docs: Documentation only changes
 - feat: A new feature
 - fix: A bug fix
 - refactor: A code change that neither fixes a bug nor adds a feature
 
 ### Branching
 
-Branch names should be snake_case. Which means that all the text must be lowercase and replace spaces with dashes. Also,
-we should add as a prefix based on the type of implementation. For example:
+Branch names should be snake_case. Which means that all the text must be
+lowercase and replace spaces with dashes. Also, we should add as a prefix based
+on the type of implementation. For example:
+
 ```
 refactor/modify_base_problem
 feature/implement_dummy_estimator
 fix/algorithm_parameter
 ```
 
 ### Pull request
+
 1. Only create pull requests to the `develop` branch.
-2. Fulfill the template
+1. Fulfill the template
```

Comment:
Sorry, this was produced by my [formatter ](https://mdformat.readthedocs.io/en/stable/users/style.html#ordered-lists). Now I made the changes and configured it properly so this will not happen again.